### PR TITLE
chore: remove `lodash-es`

### PR DESCRIPTION
**Pull request by boojack**
* remove `lodash-es`;
* update YLabel position in server side;

### CODE REVIEW

File: src/components/RepoInputer.vue [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
 <script lang="ts" setup>
 import { computed, onMounted, reactive, ref, watch } from "vue";
-import { first } from "lodash-es";
+import { first } from "lodash";
```
Comment (Megrax) [inline src/components/RepoInputer.vue:85]: Have you tested if this works? Seems lodash has renamed it to [_.head](https://lodash.com/docs/4.17.15#head). I'm not sure if `first` still works. Change it to `head` may be better.

**Review by Megrax — CHANGES_REQUESTED** [review]

File: src/components/RepoInputer.vue (hunk 1 above)
Comment (boojack) [reply src/components/RepoInputer.vue]: SGTM, updated.

**Comment by boojack** [comment]
PTAL

**Review by Megrax — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**